packages/@okta/i18n/src/properties/login.properties

```diff
@@ -1322,6 +1322,8 @@ oie.invalid.recovery.token = The recovery token is invalid.
 ## Consent
 oie.consent.scopes.admin.title = would like to access:
 oie.consent.scopes.enduser.title = would like to:
+oie.consent.scopes.granular.title = <$1>{0}</$1> <$2>requests access to</$2>
```

Comment:
Understood about the username been in a completely different view. One day we'll try to fix it

This is good, thank you! Just add one comment above describing what `{0}` means (this is for translators to have some context, there are a few examples in this file). Something like
```
# {0} is an app name
```